src/errors/index.ts

```diff
@@ -1,5 +1,7 @@
-export * from './codes';
+export * from './general-error';
```

Comment:
Extracted codes to type to remove circular import. `./types` should import nothing.